Content.Client/Launcher/LauncherConnectingGui.xaml.cs

```diff
@@ -38,14 +39,24 @@ public sealed partial class LauncherConnectingGui : Control
             _cfg = config;
 
             RobustXamlLoader.Load(this);
+            IoCManager.InjectDependencies(this);
 
             LayoutContainer.SetAnchorPreset(this, LayoutContainer.LayoutPreset.Wide);
 
             Stylesheet = IoCManager.Resolve<IStylesheetManager>().SheetSpace;
 
             ChangeLoginTip();
-            ReconnectButton.OnPressed += ReconnectButtonPressed;
-            RetryButton.OnPressed += ReconnectButtonPressed;
+            RetryButton.OnPressed += _ => _state.RetryConnect();
+            ReconnectButton.OnPressed += _ =>
```

Comment:
Why are you changing the code for the reconnect buttons?

Reply:
I think at the time which was late, I thought it was going to read better but looking at it now it is pain. Changed it back.